src/modules/changeTemperature.js

```diff
@@ -0,0 +1,17 @@
+function changeTemperature(data) {
```

Comment:
- [ ] It uses ES6 syntax
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png). 

Let's make sure to convert all the functions to arrow function syntax.